packages/story-editor/src/components/autoSaveHandler/index.js

```diff
@@ -27,36 +26,23 @@ import { useConfig, useHistory, useStory } from '../../app';
 import useIsUploadingToStory from '../../utils/useIsUploadingToStory';
 
 function AutoSaveHandler() {
-  const improvedAutosaves = useFeature('improvedAutosaves');
   const { autoSaveInterval } = useConfig();
   const {
     state: { hasNewChanges },
   } = useHistory();
-  const { isDraft, saveStory, autoSave } = useStory(
```

Comment:
@spacedmonkey @swissspidy @miina 

I got paused on this PR as I'm working on the video segmentation PR.

The last test I did --- this wasn't working as expected 

Thinking it's something with removing the isDraft --- but didn't get a chance to dig in.

Any thoughts ?

Reply:
Can you elaborate on "this wasn't working as expected"?

I do see autosave requests being triggered in the Network tab, for both drafts and published stories. So what isn't working exactly?

Reply:
Do you mean it stopped working after removing `isDraft`?

Reply:
I'll re-test ... I wasn't seeing the revisions count updating.

Reply:
🤔 --- re-tested seems to be working 

Seeing revisions and autosave network call 

Will update this and move to review.